lms/djangoapps/courseware/tests/test_video_mongo.py

```diff
@@ -803,6 +803,63 @@ def side_effect(*args, **kwargs):
 
 
 @attr('shard_1')
+class TestVideoCDNRewriting(BaseTestXmodule):
+    """
+    Tests for Video CDN.
+    """
+
+    def setUp(self, *args, **kwargs):
+        super(TestVideoCDNRewriting, self).setUp(*args, **kwargs)
+        self.original_video_file = "original_video.mp4"
+        self.original_video_url = "http://www.originalvideo.com/" + self.original_video_file
+
+    @patch.dict("django.conf.settings.CDN_VIDEO_URLS",
+                {"CN": "https://chinacdn.cn/"})
```

Comment:
https://docs.djangoproject.com/en/1.8/topics/testing/tools/#django.test.override_settings might be a clearer option here.